### PR TITLE
Add instructor notes

### PR DESCRIPTION
**Pull request by lyashevska**
closes https://github.com/esciencecenter-digital-skills/good-practices-in-research-software-development/issues/42

### CODE REVIEW

File: episodes/01-basics.md [hunk 1]
```diff
@@ -15,6 +15,18 @@ exercises: 0
 
 - What is version control and why should I use it?
 
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::: instructor
+Lesson material: https://swcarpentry.github.io/git-novice/ with our modifications.
```
Comment (svenvanderburg) [inline episodes/01-basics.md:21]:
```suggestion
Original lesson material: https://swcarpentry.github.io/git-novice/ 
```

File: episodes/02-setup.md [hunk 2]
```diff
@@ -17,6 +17,12 @@ exercises: 0
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+:::::::::::::::::::::::::::::::::::::::: instructor
+
+There are no slides for this episode. Explain what a command line is, why it is useful, and why we use it in this workshop. Participants are often new to the command line and don't get why we not use a git gui. Only focus on the bare essentials for setting up git. We shall use nano editor so that everyone is on the same page.
```
Comment (svenvanderburg) [inline episodes/02-setup.md:22]: Can you actually remove all the commands that we usually do not teach?

File: episodes/04-changes.md [hunk 3]
```diff
@@ -20,6 +20,12 @@ exercises: 0
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+:::::::::::::::::::::::::::::::::::::::: instructor
+We suggest to use slides. From slide 5 onwards:
```
Comment (svenvanderburg) [inline episodes/04-changes.md:24]: I don't agree with this suggestion to be honest. I think it is confusing. I know it works for Ole, but I think it is better to leave it to the instructor how exactly they want to use the slides. What do you think?

**Review by svenvanderburg — COMMENTED** [review]
Nice work @lyashevska! I am not sure if this will have a follow-up where we actually change the lesson material so that it conforms to some of the comments we have in the instructor notes now. At least I think that is where we should be going. See my comment on the 02-setup episode, but I think it applies to more episodes.

File: episodes/02-setup.md (hunk 2 above)
Comment (lyashevska) [reply episodes/02-setup.md:22]: Yes, I have created a separate issue for this. See https://github.com/esciencecenter-digital-skills/good-practices-in-research-software-development/issues/47

File: episodes/04-changes.md (hunk 3 above)
Comment (lyashevska) [reply episodes/04-changes.md:24]: I agree that instructors should decide for themselves which slides to show and when, I personally found it difficult to navigate in between slides and command line at the beginning, so therefore this suggestion. I can remove it as well :)

**Comment by lyashevska** [comment]
@svenvanderburg  can you please review it again and merge if happy?

**Comment by svenvanderburg** [comment]
Happy :)

**Pull request merged**